testsuite/features/step_definitions/command_steps.rb

```diff
@@ -1082,8 +1082,8 @@
   disk_path = "/tmp/#{vm_name}_disk.qcow2"
 
   # Create the throwable overlay image
```

Comment:
That comment can even be changed: it's been a long time ago there was an overlay image. now we brutally copy the template.

Reply:
It will get removed with the Salt bundle PR.